driver/config/config_test.go

```diff
@@ -1042,7 +1042,7 @@ func TestIdentitySchemaValidation(t *testing.T) {
 					for _, v := range hook.AllEntries() {
 						s, err := v.String()
 						assert.NoErrorf(t, err, "Unexpected Error")
```

Comment:
nitpick: Only use formatted errors `Error**f**` when formatting strings a la `"foo %s bar"`. Since the assertion itself already says that we assert no error, we also don't need the additional error message `Unexpected Error` as it doesn't provide additional value:

```suggestion
						require.NoError(t, err)
```